### ISSUE TITLE
allow to observe root path (only)

### ISSUE THREAD

**Issue by gullerya** [gullerya/object-observer#31]
There is a use case, when one would like to observe the changes on the root level only.

It's not possible today, since the API rejects an empty string in the `{ path: '<path>' }` option.

Should be allowed.

**Comment by gullerya** [comment]
After some playing around with this it looks that the real need is to ability to have something like `pathsOf`, which would mean observation of paths of level one of specific object.
Example, having object graph like
```
{
   user: {
      address: {
         city: 'city',
         street: 'street',
         apt: 7,
         extra: { some: 'info' }
      }
   }
}
```
call to `<>.observe(cb, { pathsOf: 'user.address' })` would observe any changes of the first level properties of `address`, but, following the example, change of property `some` of `extra` won't be notified.

**Comment by gullerya** [comment]
released in 2.9.0
